hack/modules-arm64.txt

```diff
@@ -1,6 +1,14 @@
 modules.order
```

Comment:
is the list same for both amd64 and arm64, sometimes they differ a bit

Reply:
these should be arch-independent I believe